src/editor.js

```diff
@@ -1855,8 +1855,14 @@ export class OdooEditor extends EventTarget {
         const startContainer = closestElement(range.startContainer);
         const endContainer = closestElement(range.endContainer);
 
-        const startContainerNotEditable = getClosestNotEditable(startContainer, this.editable);
-        const endContainerNotEditable = getClosestNotEditable(endContainer, this.editable);
+        const startContainerNotEditable = getHighestNotEditableFromNotEditable(
```

Comment:
"Get highest not editable from not editable" ? Wouldn't that mean the same as "Get last not editable ancestor" ?